specifyweb/frontend/js_src/lib/components/WorkBench/Template.tsx

```diff
@@ -169,13 +169,13 @@ function WbView({
                 >
                   {wbText.revert()}
                 </Button.Small>
-                <Button.Small
+                <Button.Save
                   aria-haspopup="dialog"
-                  className="wb-save"
+                  className="wb-save !py-1 !px-2"
```

Comment:
please don't hardcode `!py-1 !px-2`.
Those are defined in className.ts and should not be duplicated here.

But even besides that, there is a better solution.
Remember that Button.Small accepts a `variant` attribute?
Take a look at the code to see how you can use Button.Small, but make it look like Button.Save

<details>
  <summary>Solution</summary>
 
  ```
  <Button.Small ... variant={className.saveButton}>...</...>
  ```
</details>